presto-hive/src/main/java/io/prestosql/plugin/hive/s3/PrestoS3FileSystem.java

```diff
@@ -1255,6 +1262,14 @@ private void uploadObject()
                             break;
                     }
                 }
+                switch (s3StorageClass) {
+                    case STANDARD:
+                        request.withStorageClass(Standard);
+                        break;
+                    case IntelligentTiering:
+                        request.withStorageClass(IntelligentTiering);
+                        break;
```

Comment:
add `default` flow branch that throws some exception

Reply:
Fixed. Added a default branch and threw an exception since a default storage class is already set in the config